### PR TITLE
Orchard Framework -> Orchard Core

### PR DESCRIPTION
**Pull request by hishamco**

### CODE REVIEW

File: src/OrchardCore.Themes/TheTheme/Views/Home/Index.cshtml [hunk 1]
```diff
@@ -1,3 +1,3 @@
 <div class="message message-success">
-    <h4>@T["Welcome to the Orchard Framework, your site has been successfully set up."]</h4>
```
Comment (hishamco) [inline src/OrchardCore.Themes/TheTheme/Views/Home/Index.cshtml:2]: This is confusing while the Orchard Framework is a set of APIs that could be used in Orchard Core CMS and any other app

Comment (Skrypt) [reply src/OrchardCore.Themes/TheTheme/Views/Home/Index.cshtml:2]: Welcome to Orchard Core. You can drop the "the".

**Comment by agriffard** [comment]
the Orchard Core => Orchard Core
as The Facebook became Facebook ;-)

**Comment by hishamco** [comment]
We already use Orchard Core in the dashboard as CMS, but when I saw this in SAAS I felt it's weird

**Comment by agriffard** [comment]
`drop the "the"`
The day I can pronounce this sentence correctly, I'll consider myself as an English speaker.

**Comment by agriffard** [comment]
https://translate.google.com/?sl=en&tl=fr&text=drop%20the%20%22the%22&op=translate

**Comment by hishamco** [comment]
J'ai laissé tomber le "le"  :)

**Comment by hishamco** [comment]
Believe it or not I didn't see "The" before ;) seems because I have a cold ;)

**Comment by Skrypt** [comment]
le le de de 😄

**Review by sebastienros — APPROVED** [review]

**Review by sebastienros — APPROVED** [review]

**Pull request merged**